services/frontend-service/src/ui/components/CommitInfo/CommitInfo.tsx

```diff
@@ -15,15 +15,16 @@ along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>
 Copyright 2023 freiheit.com*/
 
 import { TopAppBar } from '../TopAppBar/TopAppBar';
-import React from 'react';
 import { GetCommitInfoResponse, Event, LockPreventedDeploymentEvent_LockType } from '../../../api/api';
+import './CommitInfo.scss';
```

Comment:
this should not be necessary, because we import all scss here: https://github.com/freiheit-com/kuberpult/blob/main/services/frontend-service/src/assets/_components.scss